test/Integration/IntegrationTestFixture.cs

```diff
@@ -28,13 +28,9 @@ public class BaseTestFixture : IDisposable
         /// </summary>
         private Process AzuriteHost;
 
-        public BaseTestFixture(bool buildJava)
+        public BaseTestFixture()
```

Comment:
If you have time - I want to revert the changes made to split up the fixture classes since it's not necessary anymore (so just one IntegrationTestFixture in this file). If you're busy though go ahead with this and I'll fix it up myself after this gets in.

Reply:
I can remove that